src/doc/config.md

```diff
@@ -95,3 +100,5 @@ Cargo recognizes a few global environment variables to configure how it runs:
   `rustdoc` instance instead.
 * `CARGO_TARGET_DIR` - Location of where to place all generated artifacts,
   relative to the current working directory.
+
+Settings specified via config files take precedence over those specified via environment variables.
```

Comment:
Oh wow, here I am not understanding the code I've already written! This is fine in this case :)